CHANGES

```diff
@@ -3,6 +3,7 @@
 - API Addition: PointSpriteParticleBatch blending is now configurable.
 - TOOLS Features: Blending mode and sort mode can be changed in Flame particle 3D editor.
 - API Addition : Polygon have method setVertex (int index, float x, float y)
+- API Addition:
```

Comment:
I guess this line is either not needed or should be completed.

Reply:
No clue why it disappeared, most likely due to all the force push magic. :D